roaring/roaring.go

```diff
@@ -695,11 +699,7 @@ func (b *Bitmap) unionInPlace(others ...*Bitmap) {
 				// range that a container can store, so instead of calculating a
 				// union we can generate an RLE container that represents the entire
 				// range.
-				tContainer = &Container{
-					runs:          []interval16{{start: 0, last: maxContainerVal}},
-					containerType: containerRun,
-					n:             maxContainerVal + 1,
-				}
+				tContainer = NewContainerRun([]interval16{{start: 0, last: maxContainerVal}})
```

Comment:
I wonder if it's worth having a special function for building a full run container (to avoid the slice allocation and conversion). Probably not necessary right now, but something to keep an eye out for.

Reply:
It might well be. I think NewContainerRun was just a bit too large to inline, so the compiler probably can't do it. But it shouldn't come up wuch.